README.md

```diff
@@ -153,6 +153,10 @@ trust_return_value_nullability = false
 # Disable running `cargo fmt` on generated files
 # (defaults to false)
 disable_format = true
+# Always generate a Builder if possible. This is mostly a convenient setter as most of the 
+# time you might want the Builder to be generated. Ignoring none-desired one can still be done with `ignore_builder`
+# (defaults to false)
+always_generate_builder = true
```

Comment:
Shouldn't this be just `generate_builder`? It's the same configuration as per-object, but in the global scope.

Also it's not "always", it would still be overridden by per-object configurations setting it to `false` or not?

Reply:
> Shouldn't this be just `generate_builder`? It's the same configuration as per-object, but in the global scope.

Probably

> Also it's not "always", it would still be overridden by per-object configurations setting it to `false`?

Yes, you can still use `ignore_builder` per object.

Reply:
>  Yes, you can still use `ignore_builder` per object.

Not `generate_builder = false`?

Reply:
I can refactor it to use that instead of having a different option for exactly the same thing?

Reply:
Better to use the same name in global and per-object scope, otherwise it's just confusing :)

Reply:
I refactored the code to remove the use of `ignore_builder` and renamed the global option to `generate_builder` as well.